capstone/capweb/models.py

```diff
@@ -1 +1,15 @@
 # Create your models here.
+from django.db import models
+
+
+class Gallery(models.Model):
+    title = models.CharField(max_length=255)
+    section = models.CharField(max_length=20, default="Applications")
+    content = models.TextField() # markdown-formatted text
+    image = models.ImageField(blank=True, null=True, upload_to="static/img/gallery/")
```

Comment:
This isn't going to work on production; the static/ folder is only served live with `DEBUG = True`, and newly uploaded files won't be copied over to staticfiles/. Likewise `{% static entry.image %}` isn't going to work in the template.

This is actually pretty tricky to do correctly. For starters it should look something like this:

- set `MEDIA_ROOT = os.path.join(BASE_DIR, 'media')` in settings_base
- set `MEDIA_URL = '/media/'`
- add this bit to our urls.py so the `/media/` URL works in development: https://docs.djangoproject.com/en/2.1/howto/static-files/#serving-files-uploaded-by-a-user-during-development
- update nginx to serve BASE_DIR + 'media' at the `/media/` URL in production

Then you can do `{{ entry.image.url }}` in the template to link to it.

The downside of just using nginx to serve the media folder in production is we can't scale to multiple servers, and that folder on the web server ends up holding data that isn't backed up anywhere. Ideally production would use django-storages and a remote storage, or an NFS mount, or a custom database-backed storage. We should chat with @bensteinberg about whether we feel like doing this correctly right now -- I think maybe not.